docs/data/date-pickers/adapters/adapters.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Library to manipulate dates
+---
+
+# Library to manipulate dates
+
+<p class="description">Choose which library the pickers must use to manipulate the dates.</p>
+
+## Why do you need a library ?
+
+Like most pickers, the MUI Date and Time Pickers need a library to manipulate the dates.
+It will be used to format, parse and mutate the date inside all of our components.
+
+The specificity of the MUI Date and Time Pickers is to let you choose which library you want to use for the date manipulation.
```

Comment:
Could you expand on what do you want to say with this sentence?
I failed to understand the meaning of it... 🙈

Reply:
OK the sentence is not great.
Basically I wanted to say that our competitor do not support multiple date library, so we are the best :angel:

Reply:
```suggestion
MUI Date and Time Pickers lets you choose which library **you** want to use for the date manipulation.
```
Maybe something along these lines would make it more clear and still deliver our message? 🤔